cypress/integration/api/book.test.ts

```diff
@@ -26,16 +25,11 @@ describe("test book api", () => {
       expect(response.body.id).equal(TEST_BOOK_UUID);
       expect(response.body.name).equal("Algorithms");
       expect(response.body.imageUrl).equal("https://localhost:1000/image.jpg");
-      expect(response.body.googleBooksId).equal("MTpsAQAAQBAJ");
```

Comment:
Make sure to update the openapi file to indicate that the `/api/book/{isbn}` endpoint no longer returns the Google Books data or id.